tests/test_runtime.py

```diff
@@ -922,14 +797,12 @@ def vqe_circuit(params):
             kwargs={"hub": "ibm-q-startup", "group": "ibm-q-startup", "project": "reservations"},
         )
 
-        provider = IBMQ.get_provider(hub="ibm-q-startup", group="xanadu", project="reservations")
-        delete_vqe_runner(provider=provider, program_id=program_id)
-        result = job.result()["fun"]
+        result = job.result()["optimal_value"]
 
         assert "parameters" in job.intermediate_results
 
     @pytest.mark.parametrize("shots", [8000])
-    def test_simple_hamiltonian_with_untrainable_parameters(self, token, tol, shots):
+    def _test_simple_hamiltonian_with_untrainable_parameters(self, token, tol, shots):
```

Comment:
Why did you add _ in front of the test function name?

Reply:
mistake! trying to disable it while running a different test, will remove.